src/account/Account.ts

```diff
@@ -98,4 +100,77 @@ export class Account implements IAccount {
       );
     });
   }
+
+  /**
+   * This method aims to buy rolls on behalf of the sender.
+   *
+   * @param amount - The amount of rolls to be purchased
+   * @param fee - The fee to be paid for the transaction.
+   * @returns An ITransactionDetails object. It contains the operationId on the network.
+   */
+  public async buyRolls(
+    amount: string,
+    fee: string,
```

Comment:
Why not using BigInt as desribed in https://github.com/massalabs/massa-standards/blob/main/units.md ?

Reply:
fixed